devlib/target.py

```diff
@@ -1041,6 +1041,12 @@ def adb_reboot_bootloader(self):
     def adb_root(self):
         adb_command(self.adb_name, 'root', timeout=30)
 
+    def usb_charge(self, enable=True, sysfs_path=None):
```

Comment:
Yes, I think hard-coding it for now is the best course of action. Given that there is no evidence that the path may vary at this time, it is better to keep things simple.